asn1/CommonLedgerProtocol.asn

```diff
@@ -0,0 +1,107 @@
+CommonLedgerProtocol
+DEFINITIONS
+AUTOMATIC TAGS ::=
+BEGIN
+
+IMPORTS
+    UInt8,
+    UInt32,
+    UInt64,
+    UInt128,
+    UInt256
+    FROM GenericTypes
+
+    Address,
+    Timestamp
+    FROM InterledgerTypes
+
+    InterledgerPacket
+    FROM GenericPacket
+;
+
+SideProtocolData ::= SEQUENCE OF SEQUENCE {
+  protocolName IA5String,
+  protocolData OCTET STRING
+}
+
+-- Ack and Response are the response types.
+-- When using these in a CLP packet, the requestId should match
+-- the requestId of the request they respond to.
+
+Ack ::= SideProtocolData
```

Comment:
@michielbdejong Why isn't the `Ack` defined as a `SEQUENCE` like everything else? `CustomRequest` is also just a wrapper around the `SideProtocolData` but it's still defined as another `SEQUENCE`